### PR TITLE
Fixed cross_entropy usage

### PR DESCRIPTION
**Pull request by SlipknotTN**
This modify doesn't affect the results, but it should be more correct to just use cross_entropy loss without adding log_softmax, because cross_entropy already adds it https://pytorch.org/docs/0.4.0/nn.html#cross-entropy

### CODE REVIEW

**Comment by vlomonaco** [comment]
Hi @SlipknotTN! Thanks for the fix, merging now! :+1:
